pgpubsub/triggers.py

```diff
@@ -26,6 +30,11 @@ def _build_payload(self, model):
             payload := '{{"app": "{model._meta.app_label}", "model": "{model.__name__}"}}'::jsonb;
             payload := jsonb_insert(payload, '{{old}}', COALESCE(to_jsonb(OLD), 'null'));
             payload := jsonb_insert(payload, '{{new}}', COALESCE(to_jsonb(NEW), 'null'));
+            SELECT current_setting('pgpubsub.notification_context', True) INTO notification_context_text;
```

Comment:
Again, for some reason I'm confused what pgpubsub.notification_context is here.

Reply:
`current_setting` is a built in function (see [docs](https://www.postgresql.org/docs/current/functions-admin.html#FUNCTIONS-ADMIN-SET-TABLE)) that allows to get the value of the setting. In this case the custom one named `pgpubsub.notification_context`.